cli/src/commands/content-directory/createEntity.ts

```diff
@@ -0,0 +1,170 @@
+import ContentDirectoryCommandBase from '../../base/ContentDirectoryCommandBase'
+import { flags } from '@oclif/command'
+import { Class as ContentDirectoryClass } from '@joystream/types/content-directory'
+import inquirer from 'inquirer'
+import { CLIError } from '@oclif/errors'
+import { InputParser } from '@joystream/cd-schemas'
+
+const CONTEXTS = ['Member', 'Curator', 'Lead'] as const
```

Comment:
I don't think there's a need to re-define `CONTEXT` const and `context` flag here, since they are already available in `ContentDirectoryCommandBase` (see how `content-directory:removeEntity` implements this).

I noticed you also re-define them (along with some methods like `getActor`, `getQuestionsFromClass`) in `updateEntityPropertyValues.ts`.

Anything that is reused by 2 or more commands can be put in `ContentDirectoryCommandBase` and either imported from there or accessed through `this`.